test.py

```diff
@@ -1,12 +1,38 @@
-from number.FloatNumber import FloatNumber
+# ckrawczyk
+# mrybarczyk
+
 from number.NumberFabric import NumberFabric
 from number.NumberType import NumberType
+from matrix.MyMatrix import MyMatrix
+import numpy
+import random
 
-print("test")
-
+maxRandomValue = 65536
+minRandomValue = -65536
 fabric = NumberFabric()
 numberType = NumberType.FLOAT
-number = fabric.createNumberFromType(numberType, 5, 1)
-number.print()
-number.add(FloatNumber(10, 2))
-print(number.value)
+
+def createMatrix(size, numberType):
+    a = [[None for i in range (0,size)] for j in range (0,size)]
+    x = [None for i in range (0,size)]
+    for i in range (0,size):
+        nominator = random.uniform(minRandomValue, maxRandomValue - 1)
+        number = fabric.createNumberFromType(numberType, nominator, maxRandomValue)
```

Comment:
czemu nie masz mianownika, tylko maxRandomValue?

Reply:
"(...) otrzymane jako iloraz r/2^16" - w tym wypadku r to nominator, a denominator to właśnie 2^16